doc/apimanager-reference.md

```diff
@@ -86,6 +88,14 @@ Generated using [github-markdown-toc](https://github.com/ekalinin/github-markdow
 | PodDisruptionBudgetSpec | `podDisruptionBudget` | \*PodDisruptionBudgetSpec | No | See [PodDisruptionBudgetSpec](#PodDisruptionBudgetSpec) reference | Spec of the PodDisruptionBudgetSpec part |
 | MonitoringSpec | `monitoring` | \*MonitoringSpec | No | Disabled | [MonitoringSpec](#MonitoringSpec) reference |
 
+### APIManagerMetaData
+
+| **Annotations**  | **Name** | **Default value** | **Description**                                                                  |
```

Comment:
it should be:

```
| **Annotations** | **Name** | **Default value** | **Description** |
| --- | --- | --- | --- |
```

Reply:
Apologies, these came from a plugin I didn't know I had on my IDE that reformats code, I've fixed all of the above. Thanks for checking it again :)